applications/tari_console_wallet/src/ui/app.rs

```diff
@@ -186,3 +198,31 @@ impl<B: Backend> App<B> {
         self.menu.draw(f, title_chunks[3], &self.app_state);
     }
 }
+
+#[derive(Error, Debug)]
+pub enum UiError {
```

Comment:
Why move this here?
Smaller files are easier to read than larger monolithic files.

Reply:
these structs are very small and putting them into separate files seems like an overkill at that point, once something grows in body size, then sure. I can split them back if you want though.

Reply:
It should be in a separate file because smaller files are easier to navigate and more friendly to conflicts